public/app/features/alerting/unified/api/stateHistoryApi.ts

```diff
@@ -2,12 +2,38 @@ import { DataFrameJSON } from '@grafana/data';
 
 import { alertingApi } from './alertingApi';
 
+export const LABELS_PREFIX = 'labels_';
+
+export function labelsDtoFromLabelsAsString(labels?: string): Record<string, string> {
+  // convert "{ke1=val1,key2=val2}" to { labels_key1: val1, labels_key2: val2 }
+  if (!labels) {
+    return {};
+  }
+  const result = labels
```

Comment:
Can we use `parsePromQLStyleMatcherLoose` instead? We already have some utility functions in `public/app/features/alerting/unified/utils/matchers.ts`.

Reply:
done!